appveyor.yml

```diff
@@ -58,6 +58,6 @@ install:
   - IF %dependencies%==highest composer update --prefer-dist --no-interaction --no-progress
 test_script:
   - ps: cd $Env:project_directory
-  - vendor\bin\phpunit
+  - php -dmemory_limit=2G vendor\bin\phpunit
```

Comment:
`vendor\bin\phpunit` is `.bat` file on Windows.

Use `php -dmemory_limit=2G vendor\phpunit\phpunit\phpunit`